rep-0003.rst

```diff
@@ -233,6 +233,66 @@ Buildsystem support:
 
   - build from source
 
+Lunar Loggerhead (May 2017 - May 2019)
+----------------------------------
+Required Support for:
+
+- Ubuntu Xenial (16.04)
+- Ubuntu Yakkety (16.10)
+- Ubuntu Zesty (17.04)
+
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 25
+
+Architecture supported(CI enabled for):
```

Comment:
This is not necessarily for CI but for for building. We currently only run CI on LTS:amd64 and we can run on other platforms but likely we don't want to run CI in parallel on all arch-distro platforms.  I'd suggest just dropping the (..).